### ISSUE TITLE
[9.0][mass_editing] Cannot remove sidebar button

### ISSUE THREAD

**Issue by cliffkujala** [OCA/server-tools#656]
Attempting to remove a sidebar button we added via the custom button in this App is not working.

Environment
Odoo Version 9.0e-20161211 (Enterprise Edition)
Running in Multiprocessing with workers
Ubuntu 16.04

* We created a Mass Edit for Product Template (product.template)
* Used it a few times, but now we want to remove the button from the Action menu. 
* Attempted to use the built in button for removing the button, but receive this error from Odoo:
  * Deletion of the action record failed.

Also, the remove button is distorted and not styled correctly.

See screenshots.
![screen shot 2016-12-19 at 10 28 04 am](https://cloud.githubusercontent.com/assets/7710215/21322691/7b561930-c5d7-11e6-8ea3-7c42350177ab.png)
![screen shot 2016-12-19 at 10 34 18 am](https://cloud.githubusercontent.com/assets/7710215/21322694/7b569522-c5d7-11e6-86bb-1deb8135c7d2.png)
![screen shot 2016-12-19 at 10 34 35 am](https://cloud.githubusercontent.com/assets/7710215/21322693/7b568442-c5d7-11e6-84e0-ec2954ad58b2.png)
![screen shot 2016-12-19 at 10 36 46 am](https://cloud.githubusercontent.com/assets/7710215/21322692/7b564f90-c5d7-11e6-88c8-bdf45a6fdbbf.png)

**Comment by pedrobaeza** [comment]
This is because you don't have permission for unlinking ir.values records. Do it with admin user.

**Comment by cliffkujala** [comment]
Thank you @pedrobaeza 

I had already uninstalled the App, so I can't test your solution but I trust it works.
Is it possible to elevate my user to full Admin Status, or can there only be one "Super" admin in Odoo?  Usually I prefer to keep the SuperAdmin a separate account and grant myself most all the powers I need to accomplish admin tasks like this (like not using root in Linux, but instead sudo'ing whenever necessary), but I'm running into many things I need to switch back to the original admin account I created when first getting going on Odoo.  At this point, I think it would be better if I was the Super, and the other account just be a General user to be logged in for tasks like being the Attendance Kiosk login, etc.  Perhaps this should be discussed elsewhere not on this particular GitHub Issue though.

However. This issue could still be somewhat considered open still for the improper styling on the remove button.
